### DISCUSSION TITLE
Browser monitoring syntax error

### DISCUSSION THREAD

**Discussion by surya-muthuraj0** [apache/skywalking#9782] [General]
Dear Team,

When ever I try this on the nodeJs application is shows syntax error.Please provide any helping points.

Thanks and Regards,
Surya Muthuraj.

![image](https://user-images.githubusercontent.com/114050333/195502042-6303e734-d7f6-4938-96bb-9d07f7a10151.png)

**Comment by Fine0830** [comment]
You should use this agent(https://github.com/apache/skywalking-nodejs) in your project.

**Reply by surya-muthuraj0** [reply]
For browser monitoring

**Comment by Fine0830** [comment]
I cannot reproduce your issue.
